http-gateway/web/src/containers/devices/provision-new-device.js

```diff
@@ -11,7 +11,7 @@ import { CopyBox } from '@/components/copy-box'
 import { isValidGuid } from '@/common/utils'
 
 import { getDeviceAuthCode } from './rest'
-import { messages as t } from './things-i18n'
+import { messages as t } from './devices-i18n'
 
 const NOOP = () => {}
 
```

Comment:
While I was testing this just now, I found a small mistake in the code, I want to ask you to correct it, it is in this file on lines 143., 150. and the other divs, which has the same `id` `auth-code-box`. I think you can change it to `className`, also in the `device-details.scss` file.

One more addition to that class to add could be this:
```css
.auth-code-box {
   span {
      text-align: right;
   }
}
```

Thanks!